utils/create-component/templates/react/stories_VisualTesting.js

```diff
@@ -1,11 +1,11 @@
 // React stories template
 
-module.exports = (storyPath, pascalCaseName, rootPath) => `
+module.exports = (storyPath, pascalCaseName, rootPath, unstable) => `
 /// <reference path="${rootPath}/../typings.d.ts" />
 /** @jsx jsx */
 import {jsx} from '@emotion/core';
 import {StaticStates} from '@workday/canvas-kit-labs-react-core';
-import {ComponentStatesTable, permutateProps, withSnapshotsEnabled} from '../../../../utils/storybook';
+import {ComponentStatesTable, permutateProps, withSnapshotsEnabled} from '../../../../${unstable ? '../' : ''}utils/storybook';
```

Comment:
Adjust path to storybook utils based on whether or not this is a Labs module. Labs modules sit one level deeper in the folder structure since they're placed in the `_labs` folder alongside the non-Labs modules.